docs/getting_started/benchmark_owner_demo.md

```diff
@@ -16,30 +16,30 @@ tutorial_id: benchmark
 
 ## Overview
 
-This guide will walk you through the essentials of how a user can create a benchmark using MedPerf. The main tasks can be summarized as follows:
+In this guide, you will learn how a user can use MedPerf to create a benchmark. The key tasks can be summarized as follows:
 
-1. Implement a valid workflow
-2. Develop a demo dataset
-3. Test your workflow
-4. Submitting the MLCubes to the MedPerf server
-5. Host the demo dataset
-6. Submit the benchmark to the MedPerf server
+1. Implement a valid workflow.
+2. Develop a demo dataset.
+3. Test your workflow.
+4. Submitting the MLCubes to the MedPerf server.
+5. Host the demo dataset.
+6. Submit the benchmark to the MedPerf server.
 
-We assume that you had [set up the general testing environment](setup.md).
+It's assumed that you have already set up the general testing environment as explained in the [setup guide](setup.md).
 
 {% include "getting_started/shared/before_we_start.md" %}
 
 ## 1. Implement a Valid Workflow
 
-This is accomplished by implementing three [MLCubes](../mlcubes/mlcubes.md):
+The implementation of a valid workflow accomplished by implementing three [MLCubes](../mlcubes/mlcubes.md):
```

Comment:
The implementation of a valid workflow is accomplished by implementing three [MLCubes](../mlcubes/mlcubes.md):